README.md

```diff
@@ -9,7 +9,7 @@ Looking for V1 of the .NET Core tooling?
 
 If you are looking for the v2.0 release of the .NET Core tools (CLI, MSBuild and the new csproj), head over to https://dot.net/core and download!
```

Comment:
Reword the header so it doesn't refer to "V1".  I'd suggest rewording this whole section to something like this (though I'm not sure if "Released" is the right term):

> Looking for Released .NET Core Tooling?
> --------------------------------------------
>
> The downloads linked on this page are for development builds of an upcoming update of the SDK, and are considered pre-release.  If you are looking for a Released (and supported) version of the .NET Core SDK, head over to https://get.dot.net and download!